src/code/z_elf_message.c

```diff
@@ -1,3 +1,62 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_elf_message/func_800F05C0.s")
+u16 ElfMessage_GetFirstCycleHint(GlobalContext* globalCtx) {
+    if (INV_CONTENT(ITEM_OCARINA) == ITEM_OCARINA) {
+        return 0;
+    }
+    if (CURRENT_DAY <= 0) {
+        return 0;
+    }
+    if (gSaveContext.weekEventReg[88] & 0x20) {
+        return 0;
+    }
+    if (gSaveContext.weekEventReg[79] & 0x10) {
+        if (gSaveContext.weekEventReg[8] & 0x40) {
+            return 0;
+        }
+        return 0x224;
```

Comment:
Those numbers are `textId`s? If they are and if you want you could put a small comment saying what the corresponding text says.
(Don't put a literal quote tho)

Reply:
Yeah they are `textId`s. Personally I prefer to leave those kinds of comments out and just use the spreadsheet, but can add them if others prefer.